azure/functions/http.py

```diff
@@ -79,7 +83,8 @@ def encode(cls, obj: typing.Any, *,
 
         if isinstance(obj, azf_abc.HttpResponse):
             status = obj.status_code
-            headers = dict(obj.headers)
+            headers = obj.headers
```

Comment:
This needs to be type-ified to help read the code later.

Reply:
will update